app/src/main/res/values-en/strings.xml

```diff
@@ -1365,19 +1365,19 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="street_parking_parallel">"Parallel"</string>
     <string name="street_parking_diagonal">"Diagonal"</string>
     <string name="street_parking_perpendicular">"Perpendicular"</string>
-    <string name="street_parking_on_street">"On roadway"</string>
-    <string name="street_parking_half_on_kerb">"Partially on roadway"</string>
-    <string name="street_parking_on_kerb">"Adjacent to roadway"</string>
-    <string name="street_parking_street_side">"Adjacent to roadway only in dedicated spaces"</string>
-    <string name="street_parking_painted_area_only">"On roadway only in dedicated spaces"</string>
+    <string name="street_parking_on_street">"On-street with no curb"</string>
```

Comment:
Is sidewalk/verge really considered as not part of road/street?

How do you call entire thing (carriageway + shoulders + sidewalk + grass verge)?

Reply:
We don't have a word for all of those things combined that I can think of.  Roadway, shoulder/breakdown lane, sidewalks.  The grass verge is considered part of the shoulder, i.e. "hard shoulder" for the paved part and "soft shoulder" for a dirt/grass part beyond that.

Reply:
If someone just said "the road", I would interpret that as just the part that's paved, and "roadway" is the part that cars drive on (i.e. road minus the hard shoulder).

Reply:
FYI I used "roadway" because I thought this is the US-English equivalent of "carriageway" in UK-English, i.e. the asphalted part of the street with the car lanes.

Reply:
The closest term in American English would be the “road right-of-way”, but that’s somewhat technical language. ROW also has other meanings in different contexts, so be careful about reusing it for other quests.

Reply:
In such case I got confused by expecting exact match to Polish.